openfisca_web_api/loader/variables.py

```diff
@@ -98,3 +98,14 @@ def build_variables(tax_benefit_system, country_package_metadata):
         name: build_variable(variable, country_package_metadata, tax_benefit_system)
         for name, variable in tax_benefit_system.variables.items()
         }
+
+
+def build_variables_overview(variables):
```

Comment:
Same than parameters: this could be merged with the 2nd processing step of `get_variables `

Reply:
I put all the logic to build the `parameters` and `variables` in `app.py`--> it's the only place we can get access to the request object.